modules/monitoring/manifests/checks.pp

```diff
@@ -162,6 +163,29 @@
       retry_interval      => 30,
       require             => Package['jq'],
     }
+
+    $cdn_log_age_threshold_hours = $::aws_environment ? {
+      'production' => 1,
+      # Staging and Integration don't produce any logs between 22:00 and 04:00
+      # when traffic replay isn't running. Fastly doesn't send us empty files.
+      default      => 6,
+    }
+    icinga::check { 'check_cdn_log_s3_freshness_assets':
+      check_command       => "check_cdn_log_s3_freshness!-e ${::aws_environment} -l govuk_assets -c ${cdn_log_age_threshold_hours}",
+      host_name           => $::fqdn,
+      service_description => "check that Fastly logs from ${cdn_log_age_threshold_hours}h ago appear in s3://govuk-${::aws_environment}-fastly-logs/govuk_assets",
```

Comment:
Random question: do we have to be judicious about `service_description` character length? This one is fine, I'm more curious overall.

Context: Where I previously work we ran into an odd Nagios issue which I'd like to know if you folks have run into for Icinga.

I believe under the hood Nagios saves the `service_description` values to a file, but that file has a character limit. If the limit is exceeded Nagios borks with an unhelpful error message. It could've just been the version of Nagios we were using.

Reply:
Chatted about this in person and we don't think it'll be a problem.